core/src/main/java/com/graphhopper/routing/ev/GeometryEncodedValue.java

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.graphhopper.routing.ev;
+
+import com.graphhopper.storage.IntsRef;
+import com.graphhopper.util.PointList;
+
+public interface GeometryEncodedValue extends DecimalEncodedValue {
```

Comment:
The name might be confusing. It is not meant as an EncodedValue that stores the 'geometry/pillar nodes/point list', but rather a road attribute that depends on this geometry.